web/src/app/sidebar/interested-courses/component.html

```diff
@@ -1,6 +1,12 @@
 <div>
+  <div *ngIf="!isActiveTerm" class="sidebar-status p-2">
+    <div class="alert alert-warning font-weight-300">
+      The term you're viewing is not the active term. You may set it as writable, but you will lose your selections. <br /> <br />
```

Comment:
I don't think "writable" is the right word here... its not clear from this text what clicking the button will do.

Reply:
I agree, do you have any suggestions?

Reply:
Maybe something like "You are not using the current term for scheduling and cannot make any changes, you may begin using this term but you will lose your selections." and the button "Start using current term"

Reply:
Although this whole active term concept is a stopgap measure and different selections per terms will essentially deprecate this entire sub-feature and that's not too far off at this point too.

Reply:
`This term is in view-only mode. To create schedules for this term, you must set it to be active. Note that changing the active term will reset your selections.`

Reply:
If thats too long maybe cut `Note that`

Reply:
Yeah that's clearer, thanks for the suggestion.

Reply:
Alright this is how that appears on a small screen
![image](https://user-images.githubusercontent.com/1103450/54721404-56525980-4b38-11e9-9905-628cdafa2595.png)
Not too bad in my opinion.